### ISSUE TITLE
Security Overhaul SKSE - Lock Variations

### ISSUE THREAD

**Issue by xXMoksXx** [tiltedphoques/Mod-Compatibility#102]
# Mod information

The mod gives the locks their own unique model and sound effects depending on the type of chest/door.

#### Download link

https://www.nexusmods.com/skyrimspecialedition/mods/58224

And also:

https://www.nexusmods.com/skyrimspecialedition/mods/59529
https://www.nexusmods.com/skyrimspecialedition/mods/62781

#### Dependencies

http://skse.silverlock.org/
https://www.nexusmods.com/skyrimspecialedition/mods/32444
https://www.nexusmods.com/skyrimspecialedition/mods/51073

# Compatibility

During 6 hours of testing, no problems were identified

**Comment by Crowlikedream** [comment]
Yeah we found it works fine as well. Going to be a good mod to have with how we are going to play our characters in our sessions.
